### ISSUE TITLE
Pose Database

### ISSUE THREAD

**Issue by Ribosome-rbx** [hbertiche/NeuralClothSim#14]
Hi, I'm very interested in this work. It's a very good project.
I noticed that, you used AMASS dataset to compare your method with SNUG and PBNS. I have some questions:
1. Which sequences in AMASS did you use for training? Are you using the same sequences as the answer [here](https://github.com/isantesteban/snug/issues/5#issuecomment-1121197317)? 
2.  Did you use all frames in the 52 sequences for training, or just a subset of them? If it's the latter, would you be nice to share the code of data spliter?
3. How many epoches did you train your model and how long did it take in your case?

Thank you again for the amazing works, and looking for ward to your reply!!!!

**Comment by hbertiche** [comment]
Thanks for your interest in my work!

I trained with the same data used in SNUG for a fair comparison.

If I recall properly, I trained using all the frames of the training sequences.

Unfortunately, I do not remember how long I trained this for. I do know that complex garments, specially loose garments, will take significantly longer to converge. Nonetheless, given the reduced size of the dataset for these experiments, it should not be a very long training either.
